src/components/BookCard/index.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import PropTypes from "prop-types";
+import Image from "../Image";
 
 function BookCard(props) {
   return (
     <div key={props.id} className="bookCard">
-      {props.imgUrl && <img src={props.imgUrl} alt={props.title} />}
+      <Image src={props.imgUrl} />
```

Comment:
The original image checked whether props.imgUrl existed before it tries to render the image, can we do that too?